### PR TITLE
docs(README.md): changed readme to new Python dependency manager

### PR DESCRIPTION
**Pull request by LeducH**
Just added some notes since you now use `poetry`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,3 +56,6 @@ Mass download MTG card arts using MTGPics with Scryfall as a backup source, down
 - You can choose whether to include extras in the search, this includes un-sets and special championship cards.
 - You can increase or decrease threads added per second depending on the speed of your internet.
 - You can choose the naming convention for saving the downloaded images.
+
+
+[1]: https://python-poetry.org/docs/basic-usage/
```
Comment (Investigamer) [inline README.md:61]: Need to add blank line at the end so we can pass pre-commit check

File: README.md [hunk 2]
```diff
@@ -19,10 +19,16 @@ Mass download MTG card arts using MTGPics with Scryfall as a backup source, down
 - Make a copy of this google sheet document on your account: https://docs.google.com/spreadsheets/d/1QnVoQ1gvz1N4TKnkJJ44_FHomy0gNoxZlaPSkua4Rmk (optional)
 
 # Setup - Python Version
+We now use `poetry` as package manager [link][1]
```
Comment (Investigamer) [inline README.md]: Maybe add a colon after manager here :)

File: README.md [hunk 3]
```diff
@@ -19,10 +19,16 @@ Mass download MTG card arts using MTGPics with Scryfall as a backup source, down
...
 - Python 3.6+
-- pip install -r requirements.txt (python dependencies)
+- clone the repository
+- install the requirements (it creates a virtual env) with
+    poetry install
 - Make a copy of this google sheet document on your account: https://docs.google.com/spreadsheets/d/1Gss4pwJZL_WzjNVFx6uDAviu1gpJdfN1fQUHbwNJl2o (optional)
-- "python app.py" to run
+- executed it with
```
Comment (Investigamer) [inline README.md]: executed > execute

**Review by Investigamer — CHANGES_REQUESTED** [review]
Thanks for the PR, if we make these changes I think we're good to go!

**Comment by LeducH** [comment]
Hey, cool looks good. 

PS: do I have to add approve some changes?

**Comment by HelixVita** [comment]
Resolves https://github.com/MrTeferi/MTG-Art-Downloader/issues/5

File: README.md [hunk 4]
```diff
@@ -16,13 +16,21 @@ Mass download MTG card arts using MTGPics with Scryfall as a backup source, down
...
+We now use `poetry` as package manager ([link][1]): 
 - Python 3.6+
-- pip install -r requirements.txt (python dependencies)
-- Make a copy of this google sheet document on your account: https://docs.google.com/spreadsheets/d/1Gss4pwJZL_WzjNVFx6uDAviu1gpJdfN1fQUHbwNJl2o (optional)
-- "python app.py" to run
+- clone the repository
+- install the requirements (it creates a virtual env) with
+```
+    poetry install
+```
```
Comment (HelixVita) [inline README.md]:
```suggestion
- Download this repository `git clone https://github.com/MrTeferi/MTG-Art-Downloader`
- If you don't have Python Poetry, simply install it with `scoop install poetry` or read more at ([link][1])
- Install dependencies `poetry install`
```

**Review by HelixVita — CHANGES_REQUESTED** [review]
Not everybody knows what Python Poetry is, and pointing the user to the docs might make it seem unnecessarily complicated to install, when really it's just `scoop install poetry` and you're done.

Then again maybe not everyone uses scoop. But I figure that's exceedingly unlikely if you're a coder _and_ you're on Windows.

**Pull request merged**